docs/topics/compiler-reference.md

```diff
@@ -156,9 +156,15 @@ environment.
 
 Use a custom JDK home directory to include into the classpath if it differs from the default `JAVA_HOME`.
 
+### -Xjdk-release=version
+
+Specify the target version of the generated JVM bytecode. Limit the API of the JDK in the classpath to the specified Java version. 
+Automatically sets [`-jvm-target version`](#jvm-target-version). Works only for [OpenJDK distributions](https://openjdk.java.net/projects/jdk/18/).
```

Comment:
Why "Works only for [OpenJDK distributions" ?

Reply:
https://youtrack.jetbrains.com/issue/KT-29974

> Sadly, this is not a part of the spec, so this file is absent in other JDK distributions, in which case we should report an error/warning saying this option is only supported when compiling against OpenJDK.

Reply:
BTW: works fine with temurin JDK distribution, so even it's not part of the spec, in can be included in distribution

Reply:
Yeah, I guess it should be rephrased somehow to say that it's not guaranteed to work on all JDK distributions, without mentioning OpenJDK specifically

Reply:
Will be fixed in a separate PR https://github.com/JetBrains/kotlin-web-site/pull/3018/files